hugo/content/Installation/developer-setup.md

```diff
@@ -2,39 +2,68 @@
 title: "Developer Setup"
 date:
 draft: false
-weight: 300
+weight: 305
 ---
 
-# Developing
-
 The [Postgres-Operator](https://github.com/crunchydata/postgres-operator) is an open source project hosted on GitHub.
 
-Developers that wish to build the Operator from source or contribute to the project via pull requests would set up a development environment through the following steps.
+This guide is intended for those wanting to build the Operator from source or contribute via pull requests.
+
+
+# Prerequisites
+The target development host for these instructions is a CentOS 7 or RHEL 7 host. Others operating systems
+are possible, however we do not support building or running the Operator on others at this time.
+
+## Environment Variables
+
+The following environment variables are expected by the steps in this guide:
 
-## Create Kubernetes Cluster
-We use either OpenShift Container Platform or kubeadm to install development clusters.
+Variable | Example | Description
+-------- | ------- | -----------
+`GOPATH` | $HOME/odev | Golang project directory 
+`PGOROOT` | $GOPATH/src/github.com/crunchydata/postgres-operator | Operator repository location 
+`PGO_BASEOS` | centos7 | Base OS for container images 
+`PGO_CMD` | kubectl | Cluster management tool executable
+`PGO_IMAGE_PREFIX` | crunchydata | Container image prefix 
+`PGO_OPERATOR_NAMESPACE` | pgo | Kubernetes namespace for the operator 
+`PGO_VERSION` | 4.0.0 | Operator version 
```

Comment:
I know it's an 'example', but is it worth updating to the current version just for the heck of it?

Reply:
Bah, I meant to - thanks for pointing out it was missing.